### PR TITLE
Update widget catalogue

### PR DESCRIPTION
**Pull request by ajdapretnar**
Update the outdated widget catalogue.

### CODE REVIEW

**Comment by ajdapretnar** [comment]
@robertcv Could you confirm this is ok? I can merge afterwards (I wanted to add you as a contributor, but I can't).

File: homepage/static/widgets-bioinformatics.json [hunk 1]
```diff
@@ -32,30 +26,12 @@
...
-        "keywords": [],
-        "text": "GenExpress"
-      },
-      {
-        "doc": "http://orange-bioinformatics.readthedocs.io/en/latest/widgets/geodatasets.html",
-        "img": "http://orange-bioinformatics.readthedocs.io/en/latest/_images/geo-data-sets.png",
-        "keywords": [],
-        "text": "GEO Data Sets"
-      },
+//      {
```
Comment (robertcv) [inline homepage/static/widgets-bioinformatics.json]: Comments are not allowed in JSON.

**Comment by ajdapretnar** [comment]
@robertcv Should be fixed. I added also the new widgets into the docs, since they will be available on RTD soon. 🤞

**Comment by robertcv** [comment]
Looks good.

**Pull request merged**